openstates/de/committees.py

```diff
@@ -22,29 +22,41 @@ def scrape(self, chamber, term):
 
         url = urls[chamber] % (session,)
```

Comment:
Actually, there is [at least one more `joint` committee](http://legis.delaware.gov/legislature.nsf/lookup/Bond_home?open). And [according to Ballotpedia](http://ballotpedia.org/Delaware_General_Assembly#Joint_committees) the Sunset Committee is actually `joint`, although we currently successfully scrape it separately for `upper` and `lower`.